skopeo inspect command - introduce a way to skip querying all available tags (#1480)
containers/skopeo: docs/skopeo-inspect.1.md

The password to access the registry.

> **--no-tags**, **-n**=_bool_

Reviewer: Just noticed on the page that the options are not listed in alpha order.  While you're in there, could you rearrange them please?

Author: Hello @TomSweeneyRedHat, 

I had a look at some other man files too and they too don't have it in alphabetical order. Given that, do you still want this to be changed for this doc content in this PR? If yes, let me know and I'll go ahead and do it.

> Query and include in the output all the available tags for the image. Defaults to `true`.

Author: The updated PR has the suggested change, although it doesn't exactly match the previous sentence. So feel free to suggest any additional change if you think is needed.

> Do not list in the output, all tags available in the same repository. Defaults to `false` because of which the output will include all tags available.

Reviewer: ```suggestion
Do not list the available tags from the repository in the output. When invoked, the `RepoTags` array will be empty.  Defaults to `false`, which includes all available tags.
```

Author: Done. The updated version of this PR (almost) uses this text. The only minor change I additionally did was, instead of saying "When invoked...", I used "When `true`...". I hope that's fine.

## EXAMPLES

> To review information for the image fedora from the docker.io registry:

Author: Done. Added the example.
